unified-codes/libs/ui/src/lib/components/inputs/molecules/ToggleButtonGroup.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import { ThemeProvider } from '@material-ui/core/styles';
+import theme from '../../../../../../../apps/web/src/muiTheme';
```

Comment:
I've done something like this elsewhere and expected someone to ping me for it in a code review 😉  Not sure the best way to make this more elegant. will think on it.